ethcore/res/ethereum/ropsten.json

```diff
@@ -2231,262 +2390,859 @@
-		"00000000000000000000000000000000000000f7": { "balance": "0" },
-		"00000000000000000000000000000000000000f8": { "balance": "0" },
-		"00000000000000000000000000000000000000f9": { "balance": "0" },
-		"00000000000000000000000000000000000000fa": { "balance": "0" },
-		"00000000000000000000000000000000000000fb": { "balance": "0" },
-		"00000000000000000000000000000000000000fc": { "balance": "0" },
-		"00000000000000000000000000000000000000fd": { "balance": "0" },
-		"00000000000000000000000000000000000000fe": { "balance": "0" },
-		"00000000000000000000000000000000000000ff": { "balance": "0" },
-		"874b54a8bd152966d63f706bae1ffeb0411921e5": { "balance": "1000000000000000000000000000000" }
+		"0x0000000000000000000000000000000000000000": {
+			"balance": "0x1"
+		},
+		"0x0000000000000000000000000000000000000001": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "ecrecover",
+				"pricing": {
+					"linear": {
+						"base": 3000,
+						"word": 0
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000002": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "sha256",
+				"pricing": {
+					"linear": {
+						"base": 60,
+						"word": 12
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000003": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "ripemd160",
+				"pricing": {
+					"linear": {
+						"base": 600,
+						"word": 120
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000004": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "identity",
+				"pricing": {
+					"linear": {
+						"base": 15,
+						"word": 3
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000005": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "modexp",
+				"activate_at": "0x19f0a0",
+				"pricing": {
+					"modexp": {
+						"divisor": 20
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000006": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "alt_bn128_add",
+				"activate_at": "0x19f0a0",
+				"pricing": {
+					"linear": {
+						"base": 500,
+						"word": 0
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000007": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "alt_bn128_mul",
+				"activate_at": "0x19f0a0",
+				"pricing": {
+					"linear": {
+						"base": 40000,
+						"word": 0
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000008": {
+			"balance": "0x1",
+			"nonce": "0x0",
+			"builtin": {
+				"name": "alt_bn128_pairing",
+				"activate_at": "0x19f0a0",
+				"pricing": {
+					"alt_bn128_pairing": {
+						"base": 100000,
+						"pair": 80000
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000009": {
+			"balance": "0x1"
+		},
+		"0x000000000000000000000000000000000000000a": {
+			"balance": "0x0"
+		},
```

Comment:
What's the reason we specify zero balance accounts? Does this even modify the state?

Reply:
The address is counted as "exists" from beginning (so it does modify the state).